src/Lykke.Common.ApiLibrary/Swagger/XmsEnum/XmsEnumOperationFilter.cs

```diff
@@ -37,13 +39,44 @@ public void Apply(Operation operation, OperationFilterContext context)
 Expected parameter source: {invalidParameter?.Source.DisplayName}";
                 throw new InvalidOperationException(message);
             }
+            
+            foreach (var parameter in GetEnumPerameters(context))
+            {
+                var operationParameter = operation.Parameters.Single(p => p.Name == parameter.Name);
+
+                XmsEnumExtensionApplicator.Apply(operationParameter.Extensions, parameter.Type, _options);
+            }
+        }
+
+        private static IEnumerable<(Type Type, string Name)> GetEnumPerameters(OperationFilterContext context)
+        {
+            return context
+                .ApiDescription
+                .ParameterDescriptions
+                .Select(p => (Type: TryGetEnumType(p), Name: p.Name))
+                .Where(x => x.Type != null);
+        }
 
-            foreach (var parameterDescription in context.ApiDescription.ParameterDescriptions.Where(p => p.Type.GetTypeInfo().IsEnum))
+        private static Type TryGetEnumType(ApiParameterDescription parameter)
```

Comment:
Doesn't look like "Try~" methods. I would name it without "Try" prefix.

Reply:
I prefer to name a mehod TryXXX, if it can return null instead of instance